docs/tutorials/tutorial-update-data.md

```diff
@@ -23,161 +23,236 @@ sidebar_label: Update existing data
+  )
+) EXTEND ("timestamp" VARCHAR, "animal" VARCHAR, "number" BIGINT))
+SELECT
+  TIME_PARSE("timestamp") AS "__time",
+  "animal",
+  "number"
+FROM "ext"
+PARTITIONED BY DAY
 ```
 
-The datasource contains three rows of data with an `animal` dimension and a `number` metric.
+<details>
+<summary> View the results</summary>
 
-## Overwrite data
+| `__time` | `animal` | `number`|
+| -- | -- | -- |
+| `2024-01-02T01:01:35.000Z`| `lion`| 300 |
+| `2024-01-02T05:01:35.000Z`| `mongoose`| 737 |
+| `2024-01-02T06:01:35.000Z`| `snake`| 1234 |
+| `2024-01-02T07:01:35.000Z`| `octopus`| 115 |
+| `2024-01-03T01:01:35.000Z`| `flamingo`| 999 |
+| `2024-01-03T01:01:35.000Z`| `tiger`| 300 |
+| `2024-01-03T05:01:35.000Z`| `polecat`| 626 |
+| `2024-01-03T06:01:35.000Z`| `iguana`| 300 |
+| `2024-01-03T07:01:35.000Z`| `seahorse`| 500 |
 
-To overwrite the data, submit another task for the same interval but with different input data.
+</details>
 
-The `quickstart/tutorial/updates-overwrite-index.json` spec performs an overwrite on the `updates-tutorial` datasource.
+Note the changes in the resulting datasource:
 
-In the overwrite ingestion spec, notice the following:
-* The `intervals` field remains the same: `"intervals" : ["2018-01-01/2018-01-03"]`
-* New data is loaded from the local file, `quickstart/tutorial/updates-data2.json`
-* `appendToExisting` is set to `false`, indicating an overwrite task
+* There is now a new row called `flamingo`.
+* The `opossum` row has the value `tiger`.
+* The `skunk` row has the value `polecat`.
+* The `iguana` and `seahorse` rows have different numbers.
 
-Submit the ingestion task to overwrite the data:
+## Update a row using partial segment overshadowing
 
-```bash
-bin/post-index-task --file quickstart/tutorial/updates-overwrite-index.json --url http://localhost:8081
-```
+In Druid, you can overlay older data with newer data for the entire segment or portions of the segment within a particular partition.
+This capability is commonly referred to as [overshadowing](../ingestion/tasks.md#overshadowing-between-segments).
 
-When Druid finishes loading the new segment from this overwrite task, run the SELECT query again.
-In the new results, the `tiger` row now has the value `lion`, the `aardvark` row has a different number, and the `giraffe` row has been replaced with a `bear` row.
-
-```bash
-dsql> SELECT * FROM "updates-tutorial";
-┌──────────────────────────┬──────────┬───────┬────────┐
-│ __time                   │ animal   │ count │ number │
-├──────────────────────────┼──────────┼───────┼────────┤
-│ 2018-01-01T01:01:00.000Z │ lion     │     1 │    100 │
-│ 2018-01-01T03:01:00.000Z │ aardvark │     1 │   9999 │
-│ 2018-01-01T04:01:00.000Z │ bear     │     1 │    111 │
-└──────────────────────────┴──────────┴───────┴────────┘
-Retrieved 3 rows in 0.02s.
-```
+:::info
 
-## Combine existing data with new data and overwrite
+The overshadow relation holds only for the same time chunk and the same datasource.
```

Comment:
I don't understand this.

Reply:
I'll remove it since [Overshadowing between segments](https://druid.apache.org/docs/latest/ingestion/tasks#overshadowing-between-segments) covers it.